seed.sh

```diff
@@ -1,21 +1,7 @@
 #!/bin/bash
```

Comment:
Within this file, do we even want to keep lines 20-30? (For some reason, Github won't let me comment down there.) 

Just in terms of avoiding confusion, considering that our workflow now emphasizes production's config as the source of truth, rather than the JSON files.